src/main/resources/templates/in-app/sponsorship/v1.0.0.html

```diff
@@ -31,6 +36,7 @@ <h2>Content</h2>
             <a
```

Comment:
Similar to above, we can probably add some descriptive text and then just hyperlink it rather than use the raw https link? 

e.g. "link to medical specialty training resources for overseas applicants"

Reply:
Yeah Thanks @john12321 , I notice that for Deferral, LTFT and Sponsorship notification as well. The text may differ according to email or href contact type, and the notification type as well. So that will need to confirm with Naz and in separate PR.